.github/workflows/camunda-platform-release.yml

```diff
@@ -155,6 +155,7 @@ jobs:
             -DpushChanges=${PUSH_CHANGES} \
             -DremoteTagging=${PUSH_CHANGES} \
             -DcompletionGoals="spotless:apply" \
+            -D skipOptimize \
```

Comment:
I might be wrong, but I think the recommended syntax is without space, i.e.,
```
-DskipOptimize
```
I think in this case it's equivalent though

Reply:
AFAIK it can be a problem on windows OS. Should be good in our case